opensearch-service-domain-cdk/README.md

```diff
@@ -0,0 +1,71 @@
+# OpenSearch Service Domain CDK
+
+### Getting Started
+
+If this is your first time using CDK in this region, will need to `cdk bootstrap` to setup required CDK resources for deployment
+
+Also ensure you have configured the desired [AWS credentials](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_prerequisites), as these will dictate the region and account used for deployment
+
+### Deploying your Domain Stack
+Before deploying your Domain stack you should fill in any desired context parameters that will dictate the composition
+of your OpenSearch Service Domain
+
+This can be accomplished by simply filling in the values in the `cdk.context.json`
+
+As well as by passing these context options as options in the CDK CLI
+```
+cdk deploy --c domainName='cdk-os-service-domain' --c engineVersion="OS_1_3_6" --c dataNodeType="r6g.large.search" --c dataNodeCount=1
+```
+* Note that these context parameters can also be passed to `cdk synth` and `cdk bootstrap` commands
```

Comment:
I can pass my datatype nodes, versions, etc to ANY of these 3 commands?  What happens if I pass different ones and alternate the order?  Do they override the cdk.context.json file?

Could you please provide a bit more explanation for a reader that may have no experience w/ CDK to understand what the appropriate workflows should be.  The notes are also missing how to tear a stack down.

Reply:
Sure will add some details on using these options as mentioned and a stack tear down section